HW02_A_Graph-Fails.Rmd

```diff
@@ -33,55 +33,79 @@ What error is being thrown? How do you correct it? (hint, the error message tell
+
+q <- p + theme(legend.position = c(0.6, 0.85)) + theme(legend.direction = "horizontal")
+
+##Removing the legend title
+q + theme(legend.title = element_blank())
 
+##Removing the plot legend
+## q + theme(legend.position = 'none')
 ```
+I added the new size argument within aesthetics function (size=2). This caused the dots to be larger. To help with the legends, I have assigne the ggplot function to p. Secondly, I added theme, legend position to top, and the graph generate has the legend on the top of the graph. Changing to a c() vector makes the legend disappear. 
+
+AHHHH ok so coordinates need to be 0 < c < 1. My b. 
+
+Correctly coded for legened modicifcations. Assigned newly positioned legend to q and modified the next two steps using q. Simply remove ## from last bit of code to run program. - DS
+
+
 
 ### Graph Fail 4
 I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
 
 ```{r}
 mpg %>% 
-ggplot(mapping = aes(x = displ, y = hwy, color = drv)) + 
-  geom_point() + 
-  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point() + geom_smooth() #se = F makes it so it won't show the error in the line of fit
+
+
 ```
 
+I may have taken the easy way out. I tried to look online on how to specify. I thought I had figured out it by specifying aesthetic command in geom_smooth (I had written, geom_smooth(aes(x = displ, y = hwy)) thinking that it would plot against only those coordinates. but to no avail). The last resot option I could think of was to eliminate the color assignment, and now only one line is present. -DS
+
+
 ### Graph Fail 5
 I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
 
 Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
 
 
 ```{r}
-ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer)) + 
-  geom_boxplot() + 
-  theme(axis.text.x = element_text(angle = 45))
+v <- ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer, fill = manufacturer)) + 
+  geom_boxplot() + theme(axis.text.x = element_text(angle = 45)) + theme(axis.text.x = element_text(hjust = 1))
+
+v
 ```
 
+##Awesome, first attempt. I assigned fill = manufacturarer to fill in the boxes in the boxplot. 
 
+After some digging, looked in the "element_text()" function. I saw angle, I played with lineheight (my answer from an initial google, but values of -3, 0, 7 in lineheight provided no changes in the car names in the graph). Then I tried hjust and used 1 as my first guess. 

```

Comment:
Another option for avoiding the overlap of the x axis labels with the plot is to add a margin, for example:  theme(axis.text.x = element_text(margin = margin(15, 0, 0, 0)))
The numbers inside "margin()" refer to the spaces to the top, right, bottom, and left of the text. The default unit for these numbers is "pt" (?) but if you want a different unit size you can specify it as the fifth argument (e.g., margin(1, 1, 1, 1, "cm").